assets/helpers/tracking/guTracking.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+// ----- Campaigns ----- //
+
+const baselineTestPrefix = 'gdnwb_copts_memco_sandc_support_baseline_support';
+
+const campaigns : {
+  [string]: string[],
+} = {
+  baseline_test: [
+    `${baselineTestPrefix}_banner`,
+    `${baselineTestPrefix}_epic`,
+    `${baselineTestPrefix}_liveblog`,
+    `${baselineTestPrefix}_header_become_supporter`,
+    `${baselineTestPrefix}_header_subscribe`,
+    `${baselineTestPrefix}_side_menu_become_supporter`,
+    `${baselineTestPrefix}_side_menu_subscribe`,
```

Comment:
In the future, it would be nice to standardise this channels and create a type which validates them.